### ISSUE TITLE
Multi-AccountLogin

### ISSUE THREAD

**Issue by Reinified** [Aliucord/plugin-requests#157]
### Discord Account

Soulz#0387

### Plugin Description

So Instagram has a feature to remember your login info when you log out, then it'll pop up a pagd like this 
![Screenshot_20210815-215426_Instagram](https://user-images.githubusercontent.com/54984650/129501589-47fe96e8-2f35-40cc-b4b3-c3c2996a793d.png)

so maybe you could find a way to make something similar to this. I know AccountSwitcher is very similar to this if not exactly the same but it's deprecated. 

### Go into more detail...

So when you log out you're greeted with a menu that remembers your Discord Information (possibly skipping 2fa) and lets you login within seconds rather than having to manually type out your login information and save it to google. This would be useful for people like me who jave multiple accounts.

TL;dr Account Switcher similar to Instagram's if possible at all.

### Request Agreement

- [X] I did indeed check to make sure my plugin request is original!

**Comment by nattadasu** [comment]
I think [Masterjoona's solution \(`Accounts`\)](https://github.com/Masterjoona/AliuPlugins) (and yes, this is the new one) using token for login multiple account is good enough? It just lack the UI thou, but at least, you can do it by command. 🤔

**Comment by hjroh0315** [comment]
only thing that it lacks- extracting the token for the current user.

**Comment by BobIsMyManager** [comment]
https://github.com/litleck/aliucord-plugins/raw/builds/Token.zip

**Comment by Reinified** [comment]
zt is working on an account switcher plugin so i can close this.
